neurots/extract_input/input_parameters.py

```diff
@@ -79,19 +79,19 @@ def merged_params(data):
             }
         )
 
-    if "basal" in neurite_types:
-        input_parameters["basal"] = merged_params({"tree_type": 3})
+    if "basal_dendrite" in neurite_types:
```

Comment:
if we want to preserve compatibility here we need "basal" or "basal_dendrite"

Reply:
ah yes, I added a converter at the top and a warning